### PR TITLE
Made minor changes to sdk-metric-parameter-tracking-for-custom-jobs file

### PR DESCRIPTION
**Pull request by sudarshan-SpringML**
Made minor changes to sdk-metric-parameter-tracking-for-custom-jobs file
- [X] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [X] Follow the style and grammar rules outlined in the above notebook template.
- [X] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [X] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [X] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under `# Official Notebooks` section, pointing to the author or the author's team.
- [X] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb [hunk 1]
```diff
@@ -8,7 +8,7 @@
       },
```
Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:8]: <p>Google Cloud Notebooks -&gt; Vertex AI Workbench</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:8]: <p>Google Cloud Notebooks -&gt; Vertex AI Workbench</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

**Review by andrewferlitsch — APPROVED** [review]

**Comment by sudarshan-SpringML** [comment]
All comments resolved

**Comment by andrewferlitsch** [comment]
Something not right about the build. It tried running a mass of notebooks:

--------------------------------------------------------------------  --------  ----------  --------------------------------------------------------------------------------------------------------------
Step #4: get_started_vertex_distributed_training.ipynb                         PASSED    00:58:51    https://console.cloud.google.com/cloud-build/builds/0eadc8a2-85f2-44d0-9a34-34b70fb769e6?project=1012616486416
Step #4: get_started_vertex_training_xgboost.ipynb                             PASSED    00:06:14    https://console.cloud.google.com/cloud-build/builds/5dddaa98-3ce0-4e54-a288-b6290bd5e9ce?project=1012616486416
Step #4: get_started_with_custom_training_pipeline_components.ipynb            PASSED    01:12:07    https://console.cloud.google.com/cloud-build/builds/4080fd8a-8ede-48ea-aa78-26866a53d0ee?project=1012616486416
Step #4: get_started_with_kubeflow_pipelines.ipynb                             PASSED    00:25:44    https://console.cloud.google.com/cloud-build/builds/ddb1d5a0-80d3-41f3-98c6-38697b535226?project=1012616486416
Step #4: sdk-metric-parameter-tracking-for-custom-jobs.ipynb                   PASSED    00:18:15    https://console.cloud.google.com/cloud-build/builds/641436e2-84f8-4e6d-9a85-b1f669823d0a?project=1012616486416
Step #4: google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb  PASSED    00:39:00    https://console.cloud.google.com/cloud-build/builds/1582dab8-8600-45c1-9403-89f18f33c45e?project=1012616486416
Step #4: tabnet_vertex_tutorial.ipynb                                          PASSED    00:19:31    https://console.cloud.google.com/cloud-build/builds/5eb40a4f-48ac-4a45-be42-92912a1f0832?project=1012616486416
Step #4: get_started_vertex_feature_store.ipynb                                FAILED    00:03:22    https://console.cloud.google.com/cloud-build/builds/32ff02c6-dfbd-4e26-8125-2653eb633407?project=1012616486416
Step #4: get_started_vertex_tensorboard.ipynb                                  FAILED    00:03:13    https://console.cloud.google.com/cloud-build/builds/5c6d01d0-251f-4348-b8a4-1f399ca71e0f?project=1012616486416
Step #4: get_started_with_automl_pipeline_components.ipynb                     FAILED    00:24:52    https://console.cloud.google.com/cloud-build/builds/99725b3b-c36a-4fc2-bdd8-ce922a0f3fd5?project=1012616486416
Step #4: get_started_with_bqml_pipeline_components.ipynb                       FAILED    00:06:13    https://console.cloud.google.com/cloud-build/builds/44269f9d-f822-4f8c-b33c-41f437fdafdb?project=1012616486416
Step #4: get_started_with_dataproc_serverless_pipeline_components.ipynb        FAILED    00:05:45    https://console.cloud.google.com/cloud-build/builds/ce05a021-d9ce-43e8-a900-42be8b7f48f5?project=1012616486416
Step #4: get_started_with_hpt_pipeline_components.ipynb                        FAILED    00:02:49    https://console.cloud.google.com/cloud-build/builds/faa57eef-d357-4a0a-9576-f0e509959552?project=1012616486416
Step #4: get_started_with_rapid_prototyping_bqml_automl.ipynb                  FAILED    00:03:56    https://console.cloud.google.com/cloud-build/builds/e3234fee-9fb6-4e74-a083-4b3090a9a7a0?project=1012616486416
Step #4: custom-tabular-bq-managed-dataset.ipynb                               FAILED    00:03:16    https://console.cloud.google.com/cloud-build/builds/02f1b6f3-73e1-4d4a-8df7-e9cfeb999d50?project=1012616486416
Step #4:

**Comment by sudarshan-SpringML** [comment]
Hi andrew. Do you know why execution test became neutral?

**Comment by sudarshan-SpringML** [comment]
ran fine in colab and github

**Comment by sudarshan-SpringML** [comment]
Added a small change to the file. Ran fine in colab and workbench

**Comment by andrewferlitsch** [comment]
automated tests not running. Submitter manually verified ran in Colab and Workbench. I am manually merging PR.

**Pull request merged**